larva/src/main/java/nl/nn/adapterframework/testtool/TestTool.java

```diff
@@ -2494,12 +2500,12 @@ private static int executeSenderWrite(String stepDisplayName, Map<String, Map<St
 		Map senderInfo = (Map)queues.get(queueName);
 		ISender sender = (ISender)senderInfo.get(senderType + "Sender");
 		Boolean convertExceptionToMessage = (Boolean)senderInfo.get("convertExceptionToMessage");
-		ParameterResolutionContext parameterResolutionContext = (ParameterResolutionContext)senderInfo.get("parameterResolutionContext");
+		IPipeLineSession session = (IPipeLineSession)senderInfo.get("session");
 		SenderThread senderThread;
-		if (parameterResolutionContext == null) {
+		if (session == null) {
```

Comment:
Dit klopt ook niet. Moet 1 functie zijn met session, ongeacht of hij leeg is.

Reply:
fixed